invokeai/app/invocations/prompt.py

```diff
@@ -55,3 +56,35 @@ def invoke(self, context: InvocationContext) -> PromptCollectionOutput:
             prompts = generator.generate(self.prompt, num_images=self.max_prompts)
 
         return PromptCollectionOutput(prompt_collection=prompts, count=len(prompts))
+    
+
+class PromptsFromFileInvocation(BaseInvocation):
+    '''Loads prompts from a text file'''
+    # fmt: off
+    type: Literal['prompt_from_file'] = 'prompt_from_file'
+
+    # Inputs
+    filename: str = Field(default=None, description="Filename of prompt text file")
```

Comment:
- `filename` is required, so it should not have a default value
- this is actually a file path - not a file name - so e.g. `path` is a better name